### PR TITLE
The TDF Deploys

### PR DESCRIPTION
**Pull request by tyeagg**
## About The Pull Request
TDF faction to replace the marines in campaign and other hvh gamemodes
image
things in order:
- l armor
- m armor + engi helm
- h armor
- sl armor
- b18
- flamer mod
- med(wip)
- fc helmet
- smartgunner helmet

todo: replace quick load outfits, item descs and names, shoes and glove sprites

## Why It's Good For The Game
looks better than just reusing xenonauten and shows faction identity better in hvh

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by PibeCantina** [comment]
I hate just because they will look better than som now

**Comment by PibeCantina** [comment]
Very good

**Comment by iforgotmeword** [comment]
uh sorry but uh what does TDF stand for?

btw make a reskin for B18 for the TDF if your replacing the marines in HVH
(they can get B18 power armor from assets)

File: code/datums/gamemodes/campaign/loadout_items/_TGMC/head.dm [hunk 1]
```diff
@@ -32,41 +32,56 @@
 
 
 /datum/loadout_item/helmet/standard
-	name = "M10X helmet"
+	name = "tdf helmet"
```
Comment (XSlayer300) [inline code/datums/gamemodes/campaign/loadout_items/_TGMC/head.dm:35]:
```suggestion
	name = "TDF helmet"
```
please do better than this.



i beg you

File: code/datums/gamemodes/campaign/loadout_items/_TGMC/head.dm [hunk 2]
```diff
@@ -109,9 +124,9 @@
 	wearer.equip_to_slot_or_del(new /obj/item/reagent_containers/hypospray/autoinjector/neuraline, SLOT_IN_HEAD)
 
 /datum/loadout_item/helmet/engineer
-	name = "M10X-W helmet"
+	name = "tdf-W helmet"
```
Comment (XSlayer300) [inline code/datums/gamemodes/campaign/loadout_items/_TGMC/head.dm:127]:
```suggestion
	name = "TDF-WLD helmet"
```

File: code/datums/gamemodes/campaign/loadout_items/_TGMC/head.dm [hunk 3]
```diff
@@ -32,41 +32,56 @@
...
+	name = "tdf helmet"
 	desc = "A standard TGMC combat helmet. Apply to head for best results."
-	item_typepath = /obj/item/clothing/head/modular/m10x
-	jobs_supported = list(SQUAD_MARINE, SQUAD_SMARTGUNNER)
+	item_typepath = /obj/item/clothing/head/modular/tdf
+	jobs_supported = list(SQUAD_MARINE)
 	loadout_item_flags = LOADOUT_ITEM_ROUNDSTART_OPTION|LOADOUT_ITEM_DEFAULT_CHOICE
 
+/datum/loadout_item/helmet/heavy
+	name = "heavy helmet"
```
Comment (XSlayer300) [inline code/datums/gamemodes/campaign/loadout_items/_TGMC/head.dm:42]:
```suggestion
	name = "TDF-HVY helmet"
```

File: code/datums/gamemodes/campaign/loadout_items/_TGMC/uniform.dm [hunk 4]
```diff
@@ -27,42 +27,12 @@
...
 /datum/loadout_item/uniform/marine_standard
-	name = "TGMC uniform"
+	name = "TDF uniform"
 	desc = "A standard-issue, kevlar-weaved, hazmat-tested, EMF-augmented marine uniform. You suspect it's not as robust-proof as advertised."
-	item_typepath = /obj/item/clothing/under/marine/black_vest
+	item_typepath = /obj/item/clothing/under/tdf/black_vest
 	jobs_supported = list(SQUAD_MARINE, SQUAD_SMARTGUNNER, SQUAD_LEADER)
 	loadout_item_flags = LOADOUT_ITEM_ROUNDSTART_OPTION|LOADOUT_ITEM_DEFAULT_CHOICE
 
-/datum/loadout_item/uniform/red_fatigue
```
Comment (XSlayer300) [inline code/datums/gamemodes/campaign/loadout_items/_TGMC/uniform.dm:36]: why are you removing the extra fatigues?

**Comment by FosterGrime-Heymynameslouie** [comment]
really cool, honestly could replace xenonaut entirely tbh

**Comment by Mantlecrawler** [comment]
https://www.youtube.com/watch?v=cRmES6IJk_o

**Comment by Gmet1234** [comment]
♩To save the mother earth from any alien attack♩

**Comment by SimplySmiley123** [comment]
> really cool, honestly could replace xenonaut entirely tbh

No.

**Comment by 74delta** [comment]
We finally look like the evil corporate mercs we are 
10/10 nice sprite work

**Comment by ButterChugger** [comment]
Should be an additional option within normal HvX. Hell even giving as a style line or Combat Robot armor type.

**Comment by Woker2004** [comment]
Please add these armor to HvX as well

**Comment by FlakBait00** [comment]
I'd take these over Jaeger any day.

**Review by Kuro020 — APPROVED** [review]

**Pull request closed without merging**